ukrainian/old-photo/conspiracy/call/question/wtf.md

```diff
@@ -0,0 +1,9 @@
+Вам подобається музика Майкла Джексона?
+
+Людина по ту сторону слухавки відповідає, що не чула жодної композиції цього
+виконавця. Раптово голос говорить, що не може продовжити розмову та кладе слухавку. Через декілька хвилин задзвенів телефон, той же голос запитав чи подобається вам музика виконавця, якого ви зовсім не знаєте. 
+
+Ви відповіли...
+
+[Відповідь](jackson/jackson.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.